### PR TITLE
User manual update

### PR DESCRIPTION
**Pull request by Termanty**

### CODE REVIEW

File: spec/usermanual/admins.html.erb [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+        <b>Material url</b> takes user to the online resource that educates and helps the course participant with solving
+        the course exercises.
+      </li>
+      <li>
+        <b>Source url</b> is a remote repository from which student will pull exercises as source files.
+        When this field is edited, TMC webapp will validate the url by attempting to clone the repository from the url.
+        You can find information on how to create a course repository by yourself from this <a href="customcourse.html#starting_from_a_template">guide</a>.
+      </li>
+      <li>
+        After the <b>Expires at</b> date no courses can be cloned from the expired template.
```
Comment (jamox) [inline spec/usermanual/admins.html.erb]: Lets explicitly comment whether the cloned courses can be still used after the course template has expired

File: spec/usermanual/admins.html.erb [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+   fill_in 'organization_rejected_reason', with: 'No such university exists, you can\'t fool me!'
+%>
+<%= screenshot %>
+<%
+  click_on 'Reject organization'
+  expect(page).to have_content 'Organization request was successfully rejected.'
+%>
+<%= screenshot %>
+
+<p>Requested organization will not be shown anywhere in the front page after rejection.</p>
```
Comment (jamox) [inline spec/usermanual/admins.html.erb]: Will the org still remain somehow reachable?

File: spec/usermanual/customcourse.html.erb [hunk 3]
```diff
@@ -278,6 +286,14 @@ require 'cgi' # for escapeHTML
   <h2>Personalized course progression</h2>
 
   <p>
+    <i>
+      Notice that course progression can now be configured from the webapp.
+      Configuration from the repository is only loaded initially, and will not change
+      after subsequent refreshes.
+    </i>
```
Comment (jamox) [inline spec/usermanual/customcourse.html.erb]: Do we have a test for this + is it true for both course templates and traditional courses?

Maybe the traditional courses should remain functioning as they were?

File: spec/usermanual/teachers.html.erb [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+<%= screenshot %>
+
+<%
+   click_on 'Show teachers in this organization'
+   highlight 'a:contains("Add a new teacher")'
+%>
+<%= screenshot %>
+
+  <p>
+    When you're a teacher in some organization, other teachers may join in at your initiative. Click 'Add a new teacher'.
```
Comment (jamox) [inline spec/usermanual/teachers.html.erb]: Should it be more like you may add new teachers?

File: spec/usermanual/teachers.html.erb [hunk 5]
```diff
@@ -0,0 +1,353 @@
...
+    <ul>
+      <li>Courses are shown by <b>Name</b> in IDE plugins (such as NetBeans). Name can't contain whitespaces.</li>
+      <li><b>Title</b> is a prettier version of the name that may contain whitespaces, and is shown on webapp pages.</li>
+      <li><b>Description</b> is a short piece of text to show when browsing this particular course.</li>
+      <li>
+        <b>Material url</b> takes user to the online resource that educates and helps the course participant with solving
+        the course exercises.
+      </li>
+      <li>
+        <b>Source url</b> is a remote repository from which student will pull exercises as source files.
```
Comment (jamox) [inline spec/usermanual/teachers.html.erb]: For course template courses I'd hide the source url

File: spec/usermanual/customcourse.html.erb (hunk 3 above)
Comment (FeisEater) [inline spec/usermanual/customcourse.html.erb]: teacher_sets_deadlines_spec: scenario 'Refreshing course does not overwrite deadlines set in the form'
Works for all courses at the moment.
I'm not entirely sure how we could provide setting deadlines through repository and webapp simultaneously

File: spec/usermanual/admins.html.erb (hunk 2 above)
Comment (FeisEater) [inline spec/usermanual/admins.html.erb]: We have a user story in backlog about organization reject notice reaching the organization requester

**Pull request merged**